torch/distributed/rpc/backend_registry.py

```diff
@@ -173,65 +173,65 @@ def _tensorpipe_construct_rpc_backend_options_handler(
     )
 
 
-# detect if any worker has invalid map_location configurations, and return
+# detect if any worker has invalid device_map configurations, and return
 # names of failed workers
-def _tensorpipe_check_device_maps(agent, map_locations):
-    if not map_locations:
-        return
+def _tensorpipe_check_device_maps(agent, device_maps):
+    if device_maps is None:
+        device_maps = {}
 
-    def check_one_worker(name, map_locations, all_device_counts):
+    def check_one_worker(name, device_maps, all_device_counts):
         device_count = all_device_counts[name]
-        wrong_worker_names = set(map_locations) - set(all_device_counts)
+        wrong_worker_names = set(device_maps) - set(all_device_counts)
         if wrong_worker_names:
             raise ValueError(f"Wrong worker names: {wrong_worker_names}")
         for worker_name in all_device_counts:
             remote_device_count = all_device_counts[worker_name]
-            if worker_name in map_locations:
-                map_location = map_locations[worker_name]
-                key_set = set(map_location.keys())
-                val_set = set(map_location.values())
+            if worker_name in device_maps:
+                device_map = device_maps[worker_name]
+                key_set = set(device_map.keys())
+                val_set = set(device_map.values())
                 if not all([
-                    len(map_location) == len(key_set),
-                    len(map_location) == len(val_set),  # check 1-to-1 mapping
+                    len(device_map) == len(key_set),
+                    len(device_map) == len(val_set),  # check 1-to-1 mapping
                     min(key_set) >= -1,
                     max(key_set) < device_count,  # check local range
                     min(val_set) >= -1,
```

Comment:
We said we didn't allow the CPU in the device map, right? If so, these -1 should become 0. And the check below should also go away.

Reply:
ah, right, forgot to change this part